server/models/Notification.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const notificationSchema = new mongoose.Schema({
+  userId: {
+    type: Schema.Types.ObjectId,
+    ref: "user",
+    required: true,
+  },
+  type: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    required: true,
+  },
+  read: {
+    type: Boolean,
+    default: false,
+    required: true,
+  },
+  date: {
```

Comment:
Optional: You can use mongoose timestamps here - which will create 'created' and 'update' time fields for you automatically :)

Reply:
Timestamps implemented.